devel/tilt/Portfile

```
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/tilt-dev/tilt 0.18.1 v
+revision            0
+
+categories          devel
+maintainers         nomaintainer
+platforms           darwin
+license             Apache-2
+
+description         Run Kubernetes environments locally, for local development
+
+long_description    Tilt powers multi-service development and makes sure they \
+                    behave! Run tilt up to work in a complete dev environment \
+                    configured for your team. \
+                    \
+                    Tilt automates all the steps from a code change to a new \
+                    process: watching files, building container images, and \
+                    bringing your environment up-to-date. Think docker build \
+                    && kubectl apply or docker-compose up.
+
+homepage            https://tilt.dev/
+
+checksums           rmd160  d2bd6001e581f54986011e10f44a58b68b6f2f80 \
+                    sha256  1a4cb78c4e880157d22d6b8e209d7fc70e77dfef0136929047e5eeee96b6841c \
+                    size    20524516
+
+depends_build-append \
+                    port:yarn port:nodejs12
+use_parallel_build  no
+build.cmd           "make build-js && ${go.bin} build -mod vendor ./cmd/tilt"
```

```suggestion
configure.cmd       make
configure.args      build-js

build.args          -mod vendor ${worksrcpath}/cmd/${name}
```

In the worst case scenario, where mod download has to be used, we can at least simplify the configure and build process.

Note that we can take advantage of the default build.cmd
https://github.com/macports/macports-ports/blob/12a814f20461168f3cc2c1fa7b24d0580819b739/_resources/port1.0/group/golang-1.0.tcl#L148